Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenHelper.java

```diff
@@ -106,6 +106,7 @@ public DocStructInterface changeCurrentDocstructType(DocStructInterface inOldDoc
                         try {
                             newDocstruct.addMetadata(old);
                         } catch (Exception e) {
+                            logger.error(e.getMessage(), e);
```

Comment:
Please replace this log call and following `Helper.setFehlermeldung` with `Helper.setErrorMessage` which bundles both calls.